sysdig/resource_sysdig_secure_cloud_auth_account.go

```diff
@@ -586,5 +591,13 @@ func cloudauthAccountToResourceData(data *schema.ResourceData, cloudAccount *v2.
 		}
 	}
 
+	// TODO: add an acc test with no values, with fedramp values and with ""
+	if !(cloudAccount.ProviderPartition.String() == cloudauth.ProviderPartition_name[0]) {
+		err = data.Set(SchemaProviderPartition, cloudAccount.ProviderPartition.String())
```

Comment:
is there a specific reason for choosing cloudauth.ProviderPartition_name[0] for comparing instead of the string directly?

Reply:
Actually not, updated it to use the related const